File: src/ui/mfnotif/src/app/model/order.ts

export class Order {
  id!: number;
  userId!: number;
  name!: string;
  method!: string;
  notify!: string;

  constructor(data: any = {}) {
      this.id = data.id;
      this.userId = data.userId;
      this.name = data.name || '';
      this.method = data.method || '-';
      this.notify = data.notify || 'N';
    }
}

File: src/main/java/com/mfnotif/model/Order.java
package com.mfnotif.model;

import lombok.*;
import javax.persistence.*;

@ToString
@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
@Entity
@Table(name = "orders")
public class Order {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private long id;
    @Column(name = "userid")
    private long userId;
    @Column(name = "name")
    private String name;
    @Column(name ="notify")
    private String notify;
}
